src/hooks/useContributions.ts

```diff
@@ -0,0 +1,25 @@
+import useSWR from "swr";
+
+interface PaginatedContributorsResponse {
+  readonly data: DbContribution[];
+  readonly meta: PageMetaDto;
+}
+
+const useContributions = (repoName: string, limit = "", orderBy = "") => {
```

Comment:
```suggestion
const useContributions = (repoName: string, limit = 10, orderBy = "recent") => {
```

Can we add defaults here similar to https://github.com/open-sauced/hot/blob/beta/src/hooks/useRepositoriesList.ts#L8